metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/common/LambdaExpression.java

```diff
@@ -56,9 +57,9 @@ public Object apply(List<Object> variableArgs) {
       lambdaVariables.put(variables.get(i), null);
     }
 
-    VariableResolver variableResolver = variable -> lambdaVariables.getOrDefault(variable
+    VariableResolver variableResolver = new DefaultVariableResolver(variable -> lambdaVariables.getOrDefault(variable
```

Comment:
Ok, help me out here, can you give me an example of the impact of this change?  Maybe a `MAP()` function and how it acts before the change and how it would act after the change?